srcalc/src/test/java/gov/va/med/srcalc/db/ResultsDaoIT.java

```diff
@@ -141,6 +150,7 @@ private void populateSampleResults()
         fResultsDao.persistSignedResult(fSampleResult2);
         fResultsDao.persistSignedResult(fSampleResult3);
         fResultsDao.persistSignedResult(fSampleResult4);
+        fResultsDao.persistHistoricalCalc(fHistoricalCalc5);
         simulateNewSession();
```

Comment:
Why does the setup() method only call this method? Were you anticipating other methods for setup or is it some proper form to keep the setup method as small as possible?